### PR TITLE
Update to contentful sdk 4.0

### PR DESCRIPTION
**Pull request by avareldev**

### CODE REVIEW

**Comment by avareldev** [comment]
I fixed the phpcs issues. Removing symfony/symfony was not possible for some reason.

To upgrade contentful I used the guides from https://github.com/contentful/contentful.php

File: src/Tests/CacheTagsGetterTraitTest.php [hunk 1]
```diff
@@ -72,39 +73,44 @@ public function testGetCacheTagsFull()
     {
         $this->fixture
             ->setRoutableTypes([$type = uniqid()])
-            ->setSlugField('slug');
+            ->setSlugField($slugField = 'slug');
 
         $resources = new ResourceArray(
-            [new DynamicEntry(
```
Comment (avareldev) [inline src/Tests/CacheTagsGetterTraitTest.php]: The DynamicEntry constructor is now protected...

File: src/CacheTagsGetterTrait.php [hunk 2]
```diff
@@ -37,22 +35,22 @@
...
-        if ($contentfulResult instanceof DynamicEntry) {
+        if ($contentfulResult instanceof Entry) {
             $tags[] = $contentfulResult->getId();
             $contentType = $contentfulResult->getContentType();
             $fields = $contentType->getFields() ?: [];
 
             if ((in_array($contentType->getId(), $this->getRoutableTypes()) &&
                 ($slugFieldName = $this->getSlugField()) &&
-                ($slugField = $contentfulResult->{'get' . ucfirst($slugFieldName)}()))) {
+                ($slugField = $contentfulResult->offsetGet(ucfirst($slugFieldName))))) {
```
Comment (b3nl) [inline src/CacheTagsGetterTrait.php]: Please use Array Access, magic property or get, but not the "implicit offset" from the ArrayAccess Interface.

Comment (avareldev) [reply src/CacheTagsGetterTrait.php]: I changed it to Array Access.

**Pull request merged**